Doc/includes/email-mime.py

```diff
@@ -22,7 +19,7 @@
     with open(file, 'rb') as fp:
         img_data = fp.read()
     msg.add_attachment(img_data, maintype='image',
-                                 subtype=imghdr.what(None, img_data))
+                                 subtype="jpeg")
```

Comment:
Minor quibble: we have a mix of single and double quotes here

Reply:
I was going to try to edit this to switch it to single quotes for consistency, but it doesn't look like I can do that in the web ui.

Reply:
Can you check https://github.com/python/cpython/pull/91461/files and then the ellipsis at the top right by the filename should have an "Edit file" link?

Reply:
Thanks @hugovk that did the trick.